js/admission.js

```diff
@@ -0,0 +1,134 @@
+
+
+$(document).ready(function(){
+    const noResultBlock= document.querySelector(".noResultBlock");
+    loadData();
+    function loadData(){
+        courseData = getData();
```

Comment:
constant is not defined
```suggestion
        const courseData = getData();
```